packages/client/src/objectSet/ObjectSet.ts

```diff
@@ -94,17 +101,23 @@ export interface ObjectSet<Q extends ObjectOrInterfaceDefinition>
 
   pivotTo: <L extends LinkNames<Q>>(type: L) => ObjectSet<LinkedType<Q, L>>;
 
-  fetchOne: Q extends ObjectTypeDefinition<any>
-    ? <L extends ObjectOrInterfacePropertyKeysFrom2<Q>>(
+  fetchOne: Q extends ObjectTypeDefinition<any> ? <
+      L extends ObjectOrInterfacePropertyKeysFrom2<Q>,
+      R extends boolean,
+      S extends StrictNonNull = StrictNonNullDefault,
```

Comment:
Hmm, so if I give it `drop`, this could return an empty result right? Could we maybe just hide/restrict the drop option for `fetchOne` for now? Feels like this could get confusing